.gitignore

```diff
@@ -0,0 +1,350 @@
+﻿## Ignore Visual Studio temporary files, build results, and
```

Comment:
I think it's better to add this content in the subfolder ``TinyCsvParser/.gitignore``. We can also decide to move the ``.gitignore`` to top-level, I just did this for my aesthetic preferences.